### ISSUE TITLE
renamed-uri-attribute-used-to-generate-aui-tag-library breaking change

### ISSUE THREAD

**Issue by gamerson** [gamerson/blade.tools#195]
https://github.com/liferay/liferay-portal/blob/master/readme/7.0/BREAKING_CHANGES.markdown#renamed-uri-attribute-used-to-generate-aui-tag-library

**Comment by lovettli** [comment]
Hey Greg,
This issue relate to jsp ,xml ,etc. i only find xml file referred to this issue on Portal. I try to update our XMLFileChecker to fix it. But our XMLFileChecker use SAX parser, it can't able to resolve attribute problem. 
I think we need to modify our XMLFileChecker to use DOM or Eclipse XML Parser to fix this issue.
What do you think?  Thanks.

**Comment by gamerson** [comment]
This is talking about JSP files Previously JSPs that used AUI library would have tags like this:

```
<%@ taglib uri="http://alloy.liferay.com/tld/aui" prefix="aui" %>
```

It should be lie this:
```
<%@ taglib uri="http://liferay.com/tld/aui" prefix="aui" %>
```

So we should be able to get this with the jsp file checker.

**Comment by lovettli** [comment]
ok, got it.

**Comment by gamerson** [comment]
Closed by 329a4b2...5abd7be
